Dockerfile

```diff
@@ -226,6 +226,7 @@ RUN \
 if [[ "${APT_DEPS_IMAGE}" == "airflow-apt-deps-ci" ]]; then \
     echo "Downloading RAT from ${RAT_URL} to ${RAT_JAR}" \
     && curl -sL ${RAT_URL} > ${RAT_JAR} \
+    && jar -tf ${RAT_JAR} \
```

Comment:
Can you add " around the variable?I would feel calmer

Reply:
Added!